dashboard/modules/metrics/dashboards/data_dashboard_panels.py

```diff
@@ -93,6 +93,18 @@
     ),
```

Comment:
do we need to update/remove the panel corresponding to `ray_data_output_bytes` above?

Reply:
This file is for the metrics tab, where we still want to show the graph